redis/src/main/java/com/hazelcast/jet/contrib/redis/RedisSinks.java

```diff
@@ -0,0 +1,593 @@
+     * @param valueFn   function that specifies how to extract stored values from incoming data items
+     * @param <T>       type of data items coming into the sink
+     * @return sink to use in {@link com.hazelcast.jet.pipeline.Pipeline#drainTo}
+     */
+    public static <T> Sink<T> sortedSet(
+            String name,
+            String uriString,
+            String set,
+            FunctionEx<T, Double> scoreFn,
+            FunctionEx<T, String> valueFn
+    ) {
+        return sortedSet(name, RedisURI.create(uriString), StringCodec::new, set, scoreFn, valueFn);
+    }
+
+    /**
+     * Creates a {@link Sink} which pushes data items into a specified Redis Sorted Set. Assumes that the sorted
+     * set identifier and stored values are all {@code String}s.
+     *
+     * @param name    name of the source being created
+     * @param uri     URI of the Redis server
+     * @param set     identifier of the Redis Sorted Set being used
+     * @param scoreFn function that specifies how to extract the scores from incoming data items
+     * @param valueFn function that specifies how to extract stored values from incoming data items
+     * @param <T>     type of data items coming into the sink
+     * @return sink to use in {@link com.hazelcast.jet.pipeline.Pipeline#drainTo}
+     */
+    public static <T> Sink<T> sortedSet(
+            String name,
+            RedisURI uri,
+            String set,
+            FunctionEx<T, Double> scoreFn,
+            FunctionEx<T, String> valueFn
+    ) {
+        return sortedSet(name, uri, StringCodec::new, set, scoreFn, valueFn);
+    }
+
+    /**
+     * Creates a {@link Sink} which pushes data items into a specified Redis Sorted Set.
+     *
+     * @param name      name of the source being created
+     * @param uriString URI of the Redis server in String form
+     * @param codecFn   supplier of {@link RedisCodec} instances, used in turn for serializing/deserializing
+     *                  keys and values
+     * @param set       identifier of the Redis Sorted Set being used
+     * @param scoreFn   function that specifies how to extract the scores from incoming data items
+     * @param valueFn   function that specifies how to extract stored values from incoming data items
+     * @param <K>       type of sorted set identifier
+     * @param <V>       type of stored values
+     * @param <T>       type of data items coming into the sink
+     * @return sink to use in {@link com.hazelcast.jet.pipeline.Pipeline#drainTo}
+     */
+    public static <K, V, T> Sink<T> sortedSet(
+            String name,
+            String uriString,
+            SupplierEx<RedisCodec<K, V>> codecFn,
+            K set,
+            FunctionEx<T, Double> scoreFn,
+            FunctionEx<T, V> valueFn
+    ) {
+        return sortedSet(name, RedisURI.create(uriString), codecFn, set, scoreFn, valueFn);
+    }
+
+    /**
+     * Creates a {@link Sink} which pushes data items into a specified Redis Sorted Set.
```

Comment:
Can we add a simple usage example per datastructure for a pipeline, example from mongodb
```
     * <p>
     * Here's an example that builds a simple source which watches all changes
     * on a collection and emits the full document of the change.
     * <pre>{@code
     * StreamSource<? extends Document> streamCollection = MongoDBSourceBuilder
     *         .stream("stream-collection",
     *                 () -> MongoClients.create("mongodb://127.0.0.1:27017"))
     *         .databaseFn(client -> client.getDatabase("dbName"))
     *         .collectionFn(db -> db.getCollection("collectionName"))
     *         .searchFn(MongoCollection::watch)
     *         .mapFn(ChangeStreamDocument::getFullDocument)
     *         .build();
     * Pipeline p = Pipeline.create();
     * StreamSourceStage<? extends Document> srcStage = p.drawFrom(streamCollection);
     * }</pre>
```
No need to add for each overloaded method, can be on the full method